components/Cards/examples/CardExampleViewController.swift

```diff
@@ -23,16 +23,15 @@ class CardExampleViewController: UIViewController {
   @IBOutlet weak var card: MDCCard!
   @IBOutlet weak var button: MDCButton!
 
-  var colorScheme = MDCSemanticColorScheme()
-  var shapeScheme = MDCShapeScheme()
-  var typographyScheme = MDCTypographyScheme()
-
-  var scheme: MDCContainerScheming {
-    let scheme = MDCContainerScheme()
-    scheme.colorScheme = colorScheme
-    scheme.typographyScheme = typographyScheme
-    scheme.shapeScheme = shapeScheme
-    return scheme
+  var containerScheme: MDCContainerScheme = MDCContainerScheme()
```

Comment:
As per the point @romoore brought up; the stored type here should ideally be `MDCContainerScheming` so that it is read-only once set.
```suggestion
  var containerScheme: MDCContainerScheming = MDCContainerScheme()
```

Reply:
Yep, I wrote this before he brought that up. Will change. Thanks.